v1/StorageSpec.dhall

```diff
@@ -2,11 +2,16 @@ let imports = ../imports.dhall
 
 let Kubernetes = imports.Kubernetes
 
-let VolumeClaimTemplate = ./VolumeClaimTemplate.dhall
+let EmbeddedPersistentVolumeClaim = ./EmbeddedPersistentVolumeClaim.dhall
 
-let StorageSpec =
-      < EmptyDir : { emptyDir : Kubernetes.EmptyDirVolumeSource.Type }
-      | VolumeClaimTemplate : { volumeClaimTemplate : VolumeClaimTemplate.Type }
-      >
-
-in  StorageSpec
```

Comment:
Why did you remove this? A `StorageSpec` need to be either an `emptyDir` or a `volumeClaimTemplate`. Please return it

Reply:
fixed